src/pages/homeAuth/HomeAuth.js

```diff
@@ -1,20 +1,16 @@
 import { useState, useEffect } from 'react';
 import { Link } from 'react-router-dom';
 import './homeAuth.scss';
+import { server } from '../../services';
 
 export const HomeAuth = ({ user }) => {
-  const [name, setName] = useState('');
   const [numCategories, setNumCategories] = useState(0);
   const [numPublished, setNumPublished] = useState(0);
   const [numProducts, setNumProducts] = useState(0);
```

Comment:
use one object, it is not necessary to keep every field from response in separate variable